UPGRADE.md

```diff
@@ -2,6 +2,14 @@
 
 ## dev-develop
 
+## Search
+
+- Adapter name changed e.g. from `massive_search_adapter.<adaptername>` to just `<adaptername>` in
+configuration.
+- Pages and snippets are now indexed in separate indexes for pages and
+  snippets. Replace all instances of `->index('content')` with
+  `->indexes(array('page', 'snippet')`.
```

Comment:
We said we don't do bullet points in the UPGRADE file anymore, please just use paragraphs instead.